src/store/actions/resource.js

```diff
@@ -477,18 +486,14 @@ export const hidePreviewResourceModalAction = () => async (dispatch) => {
 };
 
 // runs delete resource ajax
-export const deleteResourceAction = (playlistId, resourceId) => async (dispatch) => {
-  try {
-    const response = await resourceService.remove(playlistId, resourceId);
+export const deleteResourceAction = (resourceId) => async (dispatch) => {
+  const response = await resourceService.remove(resourceId);
 
-    if (response.data.status === 'success') {
-      dispatch({
-        type: actionTypes.DELETE_RESOURCE,
-        payload: { resourceId },
-      });
-    }
-  } catch (e) {
-    throw new Error(e);
+  if (response.message) {
+    dispatch({
+      type: actionTypes.DELETE_RESOURCE,
+      payload: { resourceId },
+    });
```

Comment:
Why the whole process has been changed? You just need to change param, don't you?

Reply:
playlist is nor more require while deleteing resource